PennyMe/ViewController.swift

```diff
@@ -37,9 +37,14 @@ class ViewController: UIViewController, UITextFieldDelegate, UIGestureRecognizer
     // Array for annotation database
     var artworks: [Artwork] = []
     var pinIdDict : [String:Int] = [:]
+    var isVisible : [String:Bool] = [:]
```

Comment:
I introduced this variable to keep track of which machine is visible on the map. Before that, I had issues with some machines being added multiple times, and then couldn't be removed anymore